libs/community/langchain_community/utilities/arxiv.py

```diff
@@ -225,6 +227,12 @@ def lazy_load(self, query: str) -> Iterator[Document]:
             except (FileNotFoundError, fitz.fitz.FileDataError) as f_ex:
                 logger.debug(f_ex)
                 continue
+            except HTTPError as f_ex:
```

Comment:
would it be better to instead add a `continue_on_failure: bool = False` argument to the loader the way a few other document loaders do (eg https://github.com/langchain-ai/langchain/blob/90f19028e57c5053888ad213f78d0c035169f4b9/libs/community/langchain_community/document_loaders/gcs_directory.py#L28)

Reply:
thanks a lot @baskaryan for the review. actually. I agree, it will be more practical to use such an attribute. Corrected